source/contributing/how-to-get-help.md

```diff
@@ -0,0 +1,28 @@
+# How to get help
+
+If you need assistance with one of your contributions, there are a few places you
+can go for help.
+These are sorted from specific to general, and you typically want to start as
+specific as possible, since this increases your changes to get help.
+
+Initially, you'll want to try to find individuals or groups with specific knowledge of
+your issue:
+  - look for the relevant package's maintainers, listed in its `maintainers` attribute
+  - look at the git log for the relevant files
+  - check out if any teams are responsible for the relevant subsystem:
+    - on the [NixOS website](https://nixos.org/community/#governance-teams)
+    - in the [team list file](https://github.com/NixOS/nixpkgs/blob/master/maintainers/team-list.nix)
+    - in the [`CODEOWNERS`](https://github.com/NixOS/nixpkgs/blob/master/.github/CODEOWNERS) file
+
+If that fails, you can resort to asking the community at large, using one of the official
+communication channels, such as:
+  - a related room in the [NixOS Matrix space](https://matrix.to/#/#community:nixos.org)
+  - the *Help* category on the [Discourse Forum](https://discourse.nixos.org/c/learn/9)
+    - note that people typically (but not always!) have the same username here as on
+      GitHub, so this can be another way of reaching out to specific maintainers.
```

Comment:
This should be in the above more specific section, the different ways in which you can contact specific people, Email, Discourse and Matrix should be mentioned, ideally the ones from the maintainers.nix file